utils/cross-build-mlir-air.sh

```diff
@@ -69,6 +69,7 @@ cmake .. \
     -Daarch64_TOOLCHAIN_FILE=${CMAKE_TOOLCHAIN_FILE} \
     -DLLVM_DIR=${LLVM_DIR}/build-aarch64/lib/cmake/llvm \
     -DMLIR_DIR=${LLVM_DIR}/build-aarch64/lib/cmake/mlir \
+    -DLibXAIE_ROOT=${CMAKE_SYSROOT}/opt/xaienginev2 \
```

Comment:
This I'm not so sure about, since it assume libXAIE is installed in the sysroot?

Reply:
Yes my understanding is that (i) with cross compile there must be a sysroot, and (ii) a sysroot always has an installation of libxaie under its opt dir. Are both statements true?

Reply:
libxaie doesn't have to be installed under /opt.  In fact, in mlir-aie, we cross-compile our own version of libXAIE from source, so it seems like air should just use that?

Reply:
I see. I'll update the script to allow users to pass in their own installation of libxaie.